content/en/platform/corda/4.8/open-source/key-concepts-node.md

```diff
@@ -68,35 +68,39 @@ are documented in [API: RPC operations](api-rpc.md).
 ## The service hub
 
 Internally, the node has access to a rich set of services that are used during flow execution to coordinate ledger
-updates. The key services provided are:
+updates. The key services are:
 
-* Information on other nodes on the network and the services they offer
-* Access to the contents of the vault and the storage service
-* Access to, and generation of, the node’s public-private keypairs
-* Information about the node itself
-* The current time, as tracked by the node
+* Information about other nodes on the network and the services they offer.
+* Access to the contents of the [vault](../../../../../en/platform/corda/4.8/open-source/key-concepts-vault.md) and the storage service.
```

Comment:
What is 'The storage service'?

Reply:
It's described above - the second part of the persistence layer where 'it stores transactions, attachments, and flow checkpoints'.